grandnode/grandnode2
# AuthorizationTransactionId is mapping to viewmodel

Resolves #issueNumber  
Type: **bugfix**

## Issue
AuthorizationTransactionId is not showing in the UI.

## Solution
mapped the value to viewmodel in controller.

## Breaking changes
None

## Testing
1. Place an order using any of the payment method
2. Go to order detail page
3. view payment view. verify authorization id is displaying or not.
